Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -3516,7 +3474,2305 @@
+          {
+            "type": 12,
+            "content": {
+              "version": "NotebookGroup/1.0",
+              "groupType": "editable",
+              "items": [
+                {
+                  "type": 9,
+                  "content": {
+                    "version": "KqlParameterItem/1.0",
+                    "parameters": [
+                      {
+                        "id": "c3787ab9-7fbe-4f52-9adc-47ef495bdab4",
+                        "version": "KqlParameterItem/1.0",
+                        "name": "CPUWorkProcessType",
+                        "label": "Work Process Type",
+                        "type": 2,
+                        "query": "SapNetweaver_ABAPGetWPTable_CL\r\n| distinct Typ_s",
+                        "typeSettings": {
+                          "additionalResourceOptions": [
+                            "value::1"
+                          ],
+                          "showDefault": false
+                        },
+                        "defaultValue": "value::1",
+                        "queryType": 0,
+                        "resourceType": "microsoft.operationalinsights/workspaces",
+                        "value": "UPD"
+                      }
+                    ],
+                    "style": "pills",
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces"
+                  },
+                  "name": "Utilization Parameters"
+                },
+                {
+                  "type": 3,
+                  "content": {
+                    "version": "KqlItem/1.0",
+                    "query": "let appServerList = dynamic([{CpuMemApplicationServer}]);\r\nlet workProcessType = '{CPUWorkProcessType}';\r\nSapNetweaver_ABAPGetWPTable_CL\r\n| where serverTimestamp_t between (todatetime({CpuMemTimeRange:start}) .. todatetime({CpuMemTimeRange:end}))\r\n| where SID_s == '{CpuMemSID}'\r\n| extend instanceNr_s = tostring(toint(instanceNr_d))\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 2, instanceNr_s, strcat( \"0\", instanceNr_s))\r\n| extend appServer = strcat(hostname_s, \"_\", instanceNr_s)\r\n| where array_length(appServerList) == 0 or appServer in (appServerList)\r\n| where Typ_s == workProcessType\r\n| summarize totalWP = count(), freeWP = countif(Status_s == \"Wait\") by bin(timestamp_t, 1m), Typ_s\r\n| extend activeWP = totalWP - freeWP",
+                    "size": 0,
+                    "aggregation": 3,
+                    "title": "Workprocess Utilization​",
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "visualization": "areachart",
+                    "chartSettings": {
+                      "xAxis": "timestamp_t",
+                      "yAxis": [
+                        "activeWP",
+                        "freeWP"
+                      ],
+                      "seriesLabelSettings": [
```

Comment:
so there was some feedback from Krassimir this morning about color scheme, his point was that Free worker processes should not be colored red.  Instead, Free should be colored blue and Active should be colored Red.